CHANGELOG.md

```diff
@@ -4,6 +4,43 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.2.0 (2019-01-29)
+
+This release adds bugfixes, and optimizations such as ordered append.
```

Comment:
For this section starting from "This release and up to "Notable commits" let's go with:

**This is our first Timescale-Licensed release!**

We are excited to be introducing new time-series analytical functions, advanced data lifecycle management capabilities, and improved performance. 
- **Time-series analytical functions**: Users can now use our time_bucket_gapfill function to write complex gapfilling, last object carried forward, and interpolation queries.
- **Advanced data lifecycle management**: We are introducing scheduled policies, which use our background worker framework to manage time-series data. In this release, we support scheduled `drop_chunks` and `reorder`. 
- **Improved performance**: We added support for ordered appends, which optimize a large range of queries - particularly those that contain a GROUP BY and LIMIT clause.

This release adds code under a new license, LICENSE_TIMESCALE. This code can be found in `tsl`.

^^ maybe sanity check that I didn't fudge any facts there?

Reply:
"This is our first release to include Timescale-Licensed features, in addition to new Apache-2 capabilities."

And put `time_bucket_gapfill` in backticks

Reply:
The first line `This release adds bugfixes...` should be cut.